modules/planning/common/message_process.cc

```diff
@@ -358,12 +359,19 @@ apollo::hdmap::LaneInfoConstPtr MessageProcess::GetCurrentLane(
   }
 
   for (auto& lane : lanes) {
-    for (size_t i = 0; i < routing_lane_segment_.size(); ++i) {
-      if (lane->id().id() == routing_lane_segment_[i].first) {
-        return lane;
+    for (int i = 0; i < routing_response_.road_size(); ++i) {
+      for (int j = 0; j < routing_response_.road(i).passage_size(); ++j) {
+        for (int k = 0;
```

Comment:
Loop with for-each style code, to avoid i/j/k and long row.

Reply:
I use index to loop almost everywhere for routing protobuf.  
sometimes I need the index for debugging purpose.